Automattic/wp-calypso
# A4A: Add WooCommerce extension product descriptions.

## Proposed Changes

* Adds product descriptions to the marketplace for all of the new A4A WC extensions.

## Testing Instructions

* View the product marketplace while using billing scheme 877, and ensure all product descriptions are present, accurate, and appropriate.

## Pre-merge Checklist
